### ISSUE TITLE
Revisit logobar post launch

### ISSUE THREAD

**Issue by cmajel** [cal-itp/mobility-marketplace#787]
Per @Scotchester in Figma (ty!)

 For both the scrolling logos and carousel, we should have a pause button for people to be able to stop the animation. [WCAG reference](https://www.w3.org/TR/wcag/#pause-stop-hide)

I don't think I'd classify this as a blocker to launch, but we should get to it ASAP in the next work task.

Related to: https://github.com/cal-itp/mobility-marketplace/issues/786

**Comment by leelas-pm** [comment]
@cmajel thank you so much! I think option 1 makes the most sense, happy to go with that. 

I'm open to moving it lower down, but a little skeptical - I think it's important to feature our agency partners in a prominent position. But I guess it depends on the design!
